litr/src/main/java/com/linkedin/android/litr/io/AudioRecordMediaSource.kt

```diff
@@ -61,7 +61,7 @@ class AudioRecordMediaSource(
     private var isRecording = false
 
     @Synchronized
-    fun start() {
+    fun startRecording() {
```

Comment:
I tweaked to be consistent with the new MediaSources, and also thought being more explicit (rather than start/stop) would be better.

Reply:
👍